src/components/App.tsx

```diff
@@ -28,13 +27,7 @@ const App: React.FC = () => (
         isManagedSidebar // enable this to automatically hide sidebar in mobile
         defaultManagedSidebarIsOpen={false}
       >
-        <LibRouter
-          features={
-            Config.isSingleClusterMode()
-              ? Features.SINGLE_CLUSTER_ENABLED_FEATURES
-              : Features.STANDALONE_DEPLOYMENT_ENABLED_FEATURES
-          }
-        />
+        <LibRouter features={Features.STANDALONE_DEPLOYMENT_ENABLED_FEATURES} />
```

Comment:
We should also remove `SINGLE_CLUSTER_ENABLED_FEATURES`

It doesn't seem to be referenced in UHC Portal.

Reply:
I see, these come from Assisted-ui-lib. All good.